src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -286,7 +287,19 @@ class FunSpec private constructor(
       this.modifiers += modifiers
     }
 
-    fun jvmModifiers(modifiers: Iterable<Modifier>) {
+    @JvmName("jvmModifiers")
+    @Deprecated(
+        message = "This API was missing the Builder return type and breaks chaining. " +
```

Comment:
The message here should target library users, not maintainers, something like "Deprecated. Use jvmModifiersNew()." would do.